tests/runtime/test_grpc_server.py

```diff
@@ -556,6 +577,9 @@ def test_train_fake_module_ok_response_with_datastream_csv_file(
     is_good_train_response(
         actual_response, HAPPY_PATH_TRAIN_RESPONSE, model_name, training_management_stub
     )
+    register_trained_model(
+        runtime_grpc_server, actual_response.model_name, actual_response.training_id
+    )
```

Comment:
oof, sorry about all these changes coming in from killing `autoload`

Reply:
Eh, it was the right thing to do!